batch2/batch/batch.py

```diff
@@ -74,16 +74,16 @@ def _copy_container(name, files):
                 mount_path='/gsa-key',
                 name='gsa-key')]  # FIXME: this shouldn't be mounted to every container
 
-        if self._pvc_name is not None:
+        if self.pvc_size or (self.input_files or self.output_files):
             volumes.append(kube.client.V1Volume(
-                empty_dir=kube.client.V1EmptyDirVolumeSource(
-                    size_limit=self._pvc_size),
-                name=self._pvc_name))
+                empty_dir=kube.client.V1EmptyDirVolumeSource(),
```

Comment:
you've lost the size limit. Where is that now?

Reply:
As far as I can tell, it's not used in the worker, so I'm not using it harder here.

Reply:
Ok.